components/gorgias_oauth/node_modules/@pipedream/platform

```diff
@@ -0,0 +1 @@
+../../../../node_modules/.pnpm/@pipedream+platform@1.5.1/node_modules/@pipedream/platform
```

Comment:
Are these two files within node_modules supposed to be commited? I think they should be gitignored